Tests/CorvusTests/ApplicationTests.swift

```diff
@@ -771,3 +983,30 @@ extension DatabaseID {
         .init(string: "test")
     }
 }
+
+extension XCTApplicationTester {
```

Comment:
This is a wrapper for `test()` to allow standardized testing. Currently it tests automatically for a status (200 by default) and a json response. This can also help reduce code duplication. You can see it in action in  the UpdateDTOModifier test.

Reply:
I see. Looks good!